docs/v1/P4Runtime-Spec.mdk

```diff
@@ -2900,7 +2912,13 @@ following semantics.
 - `DELETE`: Delete the group entry and deallocate the `group_id`. The group must
   not be referenced in the table action of any table entry, or the server must
   return a `FAILED_PRECONDITION` error code. If the `group_id` is invalid, the
-  server must return `NOT_FOUND`.
+  server must return `NOT_FOUND`. `group_id` is the only field which is
+  considered when performing a `DELETE` and every other field will be ignored.
+
+When setting the group membership with `INSERT` or `MODIFY`, the `members`
+repeated field must not include duplicates, &ie; members with the same
+`member_id`. The `weight` field is used instead to logically "repeat" the member
+inside the group.
```

Comment:
Would it be worth explicitly saying something like "It is explicitly allowed for the same member id to be added to multiple groups at the same time. If, as a result, an implementation 'stores' the action id and parameters in the target in multiple locations, the server must update all of those locations when a request to modify such a member is made."

Reply:
Done